code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -532,21 +532,6 @@
 	var/location = get_turf(holder.my_atom)
 	new /obj/item/slime_extract/grey(location)
 
-/datum/chemical_reaction/metalgen
-	required_reagents = list(/datum/reagent/wittel = 1, /datum/reagent/bluespace = 1, /datum/reagent/toxin/mutagen = 1)
-	results = list(/datum/reagent/metalgen = 1)
-
-/datum/chemical_reaction/metalgen_imprint
```

Comment:
hey, don't remove the metalgen imprint reaction, that's what makes the metalgen chem work

you only need to remove the reaction to CREATE metalgen. leave this reaction in in case an admin spawns in metalgen (or a botanist gets it as a strange seeds highroll).

Reply:
why the fuck is this a chem reaction and not an on_turf_apply

Reply:
because this is what imprints a metal into metalgen

in other words, you do this reaction to metalgen, and that "imprints" a material on it. at some later point, you can then splash the metalgen on something, and it'll turn it into the imprinted material (presumably in on_turf_apply(), although I haven't checked recently).

Reply:
abysmal, fixed it tho

Reply:
thanks